lib/jdk.js

```diff
@@ -61,7 +61,7 @@ exports.detect = function detect(config, opts, finished) {
 		};
 
 	function isJDK(dir) {
-		if (fs.existsSync(path.join(dir, 'bin', 'javac' + exe)) && fs.existsSync(path.join(dir, 'lib', 'dt.jar'))) {
+		if (fs.existsSync(path.join(dir, 'bin', 'javac' + exe))) {
```

Comment:
`dt.jar` doesn't seem to exist in JDK 9, jdklib uses `'java', 'javac', 'keytool', 'jarsigner'` executables in place of this check [see here](https://github.com/appcelerator/jdklib/blob/17ca66a08cc5847cff2319c032cb3d28ba9866f1/src/index.js#L102-L111), I think it might be sensible to move to that here too